impl/src/test/java/com/sun/faces/renderkit/html_basic/HtmlResponseWriterTest.java

```diff
@@ -155,6 +155,9 @@ public void testUnescapedCDATA() throws Exception {
         HtmlResponseWriter responseWriter = new HtmlResponseWriter(stringWriter, "application/xhtml+xml", "UTF-8");
         responseWriter.startElement("style", null);
         responseWriter.startCDATA();
+        // make sure empty string is handle correct
```

Comment:
Please copy the entire test method instead of adding this comment and name it testUnescapedCDataWithZeroLengthContent

Reply:
I changed the test method as requested. 
Sorry for removing the withspaces. My IDE did this and I didnot pay attention.